### PR TITLE
Add python3-distuils to required Linux package list

### LINKED ISSUE #1314: update_deps.py issue on Python 3.6.8 with Ubuntu 18.04
The script doesn't seem to run with this version of python 3 

```
# python3 ./scripts/update_deps.py
Traceback (most recent call last):
  File "./scripts/update_deps.py", line 245, in <module>
    import distutils.dir_util
ModuleNotFoundError: No module named 'distutils.dir_util'
```

Workaround for now is to edit _update_external_sources_android.sh_ to run with python2

```
--- a/build-android/update_external_sources_android.sh
+++ b/build-android/update_external_sources_android.sh
@@ -89,7 +89,7 @@ function build_shaderc () {

 # Pull down or update external dependencies
 echo "Update external dependencies based on the $ANDROIDBUILDDIR/known_good.json file"
-python3 ../scripts/update_deps.py --no-build --dir $BASEDIR --known_good_dir $BUILDDIR
+python ../scripts/update_deps.py --no-build --dir $BASEDIR --known_good_dir $BUILDDIR

 if [[ -z $nobuild ]]
 then
```

### PR DESCRIPTION
**Pull request by mark-lunarg**
Fixes #1314.

### CODE REVIEW

**Comment by johnzupin** [comment]
This package is Ubuntu 18.04 specific, it doesn't exist on 16.04.

I'm looking at 16.04 now to see if any changes need to be made to requirements.

I might request a change for a 18.04 sub-section under Linux requirements.

**Comment by mark-lunarg** [comment]
@johnzupin, the interwebs were saying python-distutils would work, is that available on 16.04?

**Comment by johnzupin** [comment]
No that doesn't exist either. It looks like the package is called python[3]-distlib on Ubunut 16.04 but I'm trying to double check. 

There is a python[3]-distutils-extra package on both distros. Let me see if that package will require the distutils packages required and we could just specify that one.

**Comment by johnzupin** [comment]
Lets change the package from python3-distutils to python3-distutils-extra. Both these packages exist on both distros, and it will require python3-distutils when installing on Ubuntu 18.04.

File: BUILD.md [hunk 1]
```diff
@@ -423,7 +423,7 @@ repository to other Linux distributions.
 
     sudo apt-get install git build-essential libx11-xcb-dev \
         libxkbcommon-dev libwayland-dev libxrandr-dev \
-        libegl1-mesa-dev
+        libegl1-mesa-dev python3-distutils
```
Comment (johnzupin) [inline BUILD.md]: Change to python3-distutils-extra

**Comment by johnzupin** [comment]
Thinking more about the problem linked at the top comment I'm wondering if this isn't the right fix for this. 

For Ubuntu 18.04, a user would need to install either the python-distutils or python3-distutils package depending on what python version they have installed.

Maybe a sub-section under "Required Package List" for Ubuntu 18.04 listing the two packages depending on the python version is a better way.

**Comment by johnzupin** [comment]
Updatedd the build.md file. This package is only required for the update_deps.py script so if it should be in another section ([Building Dependent Repositories with known_good](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/BUILD.md#building-dependent-repositories-with-known-good-revisions)?) let me know.

**Pull request merged**